misaligned/MisalignedTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace MisalignedSpace
+{
+  public class MisalignedTest
+  {
+    static void Main(string[] args)
+    {
+      List<ColorCodeInfo> result =Misaligned.printColorMap();
+      Debug.Assert(result.Count == 25);
+      Debug.Assert(result[0].printFormat.Length == result[result.Count - 1].printFormat.Length);
+      Debug.Assert(result[0].majorColor == "White");
+      Debug.Assert(result[0].minorColor == "Blue");
+      Debug.Assert(result[24].majorColor == "Violet");

```

Comment:
tests are not sufficient. it only test first and last. Also alignment of strings (printed) related tests are missing.